pkgs/development/tools/vultr/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, buildGoPackage, fetchFromGitHub }:
+
+buildGoPackage rec {
+  name = "vultr-${version}";
+  version = "${major}.${minor}.${patch}";
+  major = "1";
+  minor = "12";
+  patch = "0";
```

Comment:
The verbosity of major / minor / patch here isn't used anywhere, so a single `version = "1.12.0";` would do nicely here. These major/minor/patch break-outs can make sense sometimes, but unless there is a specific reason only makes it more difficult to update later.